Editor/Window/ManagedEC2Page.cs

```diff
@@ -0,0 +1,145 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using AmazonGameLift.Editor;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.UIElements;
+using OperatingSystem = Amazon.GameLift.OperatingSystem;
+
+namespace AmazonGameLift.Editor
+{
+    public class ManagedEC2Page
+    {
+        private readonly VisualElement _container;
+        private readonly DeploymentSettings _model;
+        private readonly Button _deployButton;
+        private readonly Button _redeployButton;
+        private readonly Button _deleteButton;
+        private readonly Button _launchClientButton;
+        private readonly DeploymentScenariosInput _deploymentScenariosInput;
+        private readonly FleetParametersInput _fleetParamsInput;
+        private readonly Label _ec2DeploymentStatusLabel;
+
+        public ManagedEC2Page(VisualElement container)
+        {
+            _container = container;
+            _model = DeploymentSettingsFactory.Create();
+            _model.Restore();
+            _model.Refresh();
+            var parameters = new FleetParameters
+            {
+                FleetName = _model.FleetName ?? $"{Application.productName}-ManagedFleet",
+                LaunchParameters = _model.LaunchParameters ?? $"",
+                BuildName = _model.BuildName ??
+                            $"{Application.productName}-{_model.ScenarioName.Replace(" ", "_")}-Build",
+                GameServerFile = _model.BuildFilePath,
+                GameServerFolder = _model.BuildFolderPath,
+                OperatingSystem = FleetParametersInput.GetOperatingSystem(_model.BuildOperatingSystem) ??
```

Comment:
minor: Is this the only spot such a conversion is required? It seems more useful to make this conversion when populating the `DeploymentSettings`. This would require the OS related utilities be moved elsewhere that is more accessible from `FleetParametersInput`, but seems like a nice to have.